docs/userGuide/developingASite.md

```diff
@@ -40,8 +40,9 @@ MarkBind will generate the site in a folder named `_site` in the current directo
 | `-f`, `--force-reload` | Force a full reload of all site files when a file is changed. |
 | `-p`, `--port <port>` | The port used for serving your website. |
 | `-s`, `--site-config <file>` | Specify the site config file (default: site.json) |
+| `--baseurl <override-base-url>`	| Build site with the baseUrl overridden. |
 | `--one-page <file>` | Render and serve only a single page from your website. |
-| --no-open | Don't open browser automatically. |
+| `--no-open` | Don't open browser automatically. |

```

Comment:
Let's move `--no-open` to be above `--one-page <file>` as well for it to be alphabetical order.

Reply:
Done!